app/views/bets/_display_bets.html.erb

```diff
@@ -5,10 +5,22 @@
         <tr>
           <th class="col-md-2">Created By</th>
           <th class="col-md-2">Created at</th>
+          <th class="col-md-2">Expires at</th>
           <th class="col-md-2">Total amount bet</th>
-          <th class="col-md-6">Bet description</th>
+          <th class="col-md-4">Bet description</th>
         </tr>
       </thead>
+      <tbody>
+        <% bets.each do |bet| %>
+          <tr>
+            <td class="col-md-2"><%= bet.creator_id %></td>
+            <td class="col-md-2"><%= bet.created_at %></td>
+            <td class="col-md-2"><%= bet.expires_at %></td>
+            <td class="col-md-2"><%= bet.bet_total %></td>
+            <td class="col-md-4"><%= bet.bet %></td>
```

Comment:
Reading this code I have no idea what `bet.bet` is. I feel like the naming could be more clear somewhere in there.